homework-1/cash_register.py

```diff
@@ -0,0 +1,49 @@
+
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+
+class CashRegister:
+
+    def __init__(self):
```

Comment:
Is the class going to be instantiated with some parameters?

Like CashRegister(balance=200)